arangod/RocksDBEngine/RocksDBEdgeIndex.cpp

```diff
@@ -648,31 +679,49 @@ static std::string FindMedian(rocksdb::Iterator* it, std::string const& start,
   return it->key().ToString();  // median is exclusive upper bound
 }
 
-void RocksDBEdgeIndex::warmup(transaction::Methods* trx,
-                              std::shared_ptr<basics::LocalTaskQueue> queue) {
+Result RocksDBEdgeIndex::scheduleWarmup() {
   if (!useCache()) {
-    return;
+    return {};
   }
 
-  // prepare transaction for parallel read access
-  RocksDBTransactionState::toState(trx)->prepareForParallelReads();
+  auto ctx =
+      transaction::V8Context::CreateWhenRequired(_collection.vocbase(), false);
```

Comment:
Why do we use a V8Context here?

Reply:
I think using a V8 context is not necessary anymore. The original code used a V8 context, because the `load indexes into memory` operation could have been called via the `db.<collection>.loadIndexesIntoMemory()` function, i.e. from within V8.
As we are now always executing the loading ops in a background thread, we could change this to use a standalone context. 
I suggest we make this change in devel first to see if it has any side-effects.